todo_service/apis/handlers/todo.go

```diff
@@ -100,6 +101,19 @@ func (th *TodoHandlers) HandleGetTodo(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, httpResponse.GetSuccessResponse(todo))
 }
 
+func (th *TodoHandlers) HandleGetTodosByDate(ctx *gin.Context) {
+	dueDate := ctx.Query("due_date")
+
+	log.GetLog().Warn(dueDate)
+	todos, err := th.todoService.GetTodosByDate(dueDate)
+	if err != nil {
+		ctx.JSON(http.StatusBadRequest, httpResponse.GetErrorResponse(err.Error()))
+		return
+	}
+
+	ctx.JSON(http.StatusOK, httpResponse.GetSuccessResponse(todos))
+}
```

Comment:
GetTodosByDate() gives incomplete information. Less than date? greater than date? Maybe the API can become a bit more generic